Assets/Samples/Volume Framework/Scripts/ToonSharpenTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace DELTation.ToonRP.PostProcessing.BuiltIn
+{
+    public class ToonSharpenTest : ToonPostProcessingPassBase
```

Comment:
I suggest merging the new logic into the existing sharpen pass.

Reply:
Let's merge the copied sharpen pass/asset to the one in the package.
Also bring the corresponding volume component to the package folder.